src/main/java/alpaca/actions/Delete.java

```diff
@@ -0,0 +1,54 @@
+package alpaca.actions;
+
+import java.util.ArrayList;
+
+import alpaca.tasks.Task;
+import alpaca.exceptions.ValueNotFound;
+import alpaca.exceptions.InvalidInput;
+
+/**
+ * Handles the processing of the Delete command
+ **/
+public abstract class Delete extends Template {
+    protected static String trigger = "delete";
+
+    protected static Boolean isTrigger(String input, String trigger) {
+        return input.toLowerCase().replaceAll(" ", "")
+                .replaceAll("[0-9]", "").equals(trigger);
+    }
+
+    /**
+     * Processes delete command
+     * 
+     * @param input The command inputted
+     * @param list  The list to delete tasks from
+     * @throws ArrayIndexOutOfBoundsException If an item with that index does not
+     *                                        exist
+     * @throws ValueNotFound                  If there is additional information
+     *                                        following the command
+     * @throws InvalidInput                   If the information following the
+     *                                        command is not comprehesible
+     **/
+    public static Boolean run(String input, ArrayList<Task> list)
```

Comment:
Consider using `boolean` instead? I noticed this being used in other method signatures as well, there's auto boxing done implicitly